src/pages/CardPage.js

```diff
@@ -0,0 +1,239 @@
+  },
+
+  {
+    id: 2,
+    icon: "far fa-clock",
+    heading: "Hours:",
+    content: "<p>Monday through Friday<br />8 a.m. to 4:30 p.m.</p>"
+  },
+  {
+    id: 3,
+    icon: "far fa-envelope",
+    heading: "Email:",
+    content: "<p><a href=`#`>publicworks@​baltimorecountymd.gov</a></p>"
+  },
+  {
+    id: 4,
+    icon: "fas fa-mobile-alt",
+    heading: "Phone:",
+    content:
+      "<p><a href=`tel:410-887-6789`>410-887-6789</a></p><p><h3>Emergency Dispatch:</h3></p><p><a href=`tel:410-887-6789`>410-887-6789</a></p>"
+  },
+  {
+    id: 5,
+    icon: "fas fa-exclamation-circle",
+    heading: "BALTCOGO311",
+    content: " <p><a href=`#`>Report a problem</a> online.</p>"
+  },
+  {
+    id: 6,
+    icon: "fas fa-user",
+    heading: "Director:",
+    content: "<p>Steve Walsh, P.E.</p>"
+  }
+];
+const SampleCard1 = () => (
+  <Card>
+    <CardContent>
+      <svg
+        width="100%"
+        height="150px"
+        xmlns="http://www.w3.org/2000/svg"
+        preserveAspectRatio="xMidYMid slice"
+        focusable="false"
+        role="img"
+        aria-label="Placeholder: Thumbnail"
+      >
+        <rect width="100%" height="200px" fill="#C2C2C2"></rect>
+      </svg>
+      <h3>Centered With Image</h3>
+      <p>.8 acres</p>
+      <p>
+        <a href="https://www.google.com/maps">
+          300 Lennox Avenue Room 202 Towson, Maryland 21286
+        </a>
+      </p>
+    </CardContent>
+    <CardFooter>
+      <button type="button" class="dg_button">
+        Learn More
+      </button>
+    </CardFooter>
+  </Card>
+);
+
+const SampleCard2 = () => (
+  <Card>
+    <CardContent>
+      <div className="dg_icon-container">
+        <i className="fas fa-star" aria-hidden="true"></i>
+      </div>
+      <h3>Centered With Icon</h3>
+      <p>.8 acres</p>
+      <p>
+        <a href="https://www.google.com/maps">
+          300 Lennox Avenue Room 202 Towson, Maryland 21286
+        </a>
+      </p>
+    </CardContent>
+    <CardFooter>
+      <button type="button" class="dg_button">
+        Learn More
+      </button>
+    </CardFooter>
+  </Card>
+);
+
+const SampleCard3 = () => (
+  <Card>
+    <CardContent className="text-left">
+      <svg
+        width="100%"
+        height="150px"
+        xmlns="http://www.w3.org/2000/svg"
+        preserveAspectRatio="xMidYMid slice"
+        focusable="false"
+        role="img"
+        aria-label="Placeholder: Thumbnail"
+      >
+        <rect width="100%" height="200px" fill="#C2C2C2"></rect>
+      </svg>
+      <h3>Align Left With Image</h3>
+      <p>.8 acres</p>
+      <p>
+        <a href="https://www.google.com/maps">
+          300 Lennox Avenue Room 202 Towson, Maryland 21286
+        </a>
+      </p>
+    </CardContent>
+    <CardFooter className="text-left">
+      <button type="button" class="dg_button">
+        Learn More
+      </button>
+    </CardFooter>
+  </Card>
+);
+
+const mainContent = () => (
+  <Section>
```

Comment:
for now can remove this section and replace with <React.Fragment>, this is causing some huge padding, something we need to resolve.

Reply:
Updated and committed.